pkg/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -23,38 +23,33 @@ import (
 	"sync"
 	"time"
 
-	"k8s.io/klog/v2"
-
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	informers "k8s.io/client-go/informers/core/v1"
-	corelisters "k8s.io/client-go/listers/core/v1"
-	"k8s.io/client-go/tools/cache"
-	"k8s.io/client-go/tools/record"
-
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/kubernetes/scheme"
 	v1core "k8s.io/client-go/kubernetes/typed/core/v1"
+	corelisters "k8s.io/client-go/listers/core/v1"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/record"
 	cloudprovider "k8s.io/cloud-provider"
+	"k8s.io/klog/v2"
 	utiltaints "k8s.io/kubernetes/pkg/util/taints"
+	netutils "k8s.io/utils/net"
 
+	"sigs.k8s.io/cloud-provider-azure/cmd/cloud-controller-manager/app/options"
```

Comment:
we should avoid importing packages from cmd packages. Please move the default values to a separate `pkg/consts` package

Reply:
sure, I would aggregate all the consts under `pkg/` to `pkg/consts`